packages/mjml-core/src/parsers/document.js

```diff
@@ -58,8 +59,14 @@ const mjmlElementParser = (elem, content) => {
 const parseHead = (head, attributes) => {
   const $container = dom.parseHTML(attributes.container)
 
-  each(compact(filter(dom.getChildren(head), child => child.tagName)), element => {
-    const handler = MJMLHeadElements[element.tagName.toLowerCase()]
+  each(compact(filter(dom.getChildren(head), child => child.tagName)), el => {
+    const element = {
+      attributes: dom.getAttributes(el),
+      children: toArray(el.childNodes),
```

Comment:
Well for now i'm ok with that it works well ! Really appreciate the help ! Thanks and have a nice holiday then 👍